custom_components/tuya_local/devices/goldair_gpph900_heater.yaml

```diff
@@ -0,0 +1,172 @@
+name: Goldair Panel heater (GPPH770/GPPH780; GPPH900/GPPH910)
+products:
+  - id: xr6jsgylldbpkaz9
+    manufacturer: Goldair
+    model: GPPH770/GPPH780; GPPH900/GPPH910
+entities:
+  - entity: climate
+    translation_key: heater
+    dps:
+      - id: 1
+        name: hvac_mode
+        type: boolean
+        mapping:
+          - dps_val: true
+            value: "heat"
+          - dps_val: false
+            value: "off"
+      - id: 4
+        name: preset_mode
+        type: string
+        mapping:
+          - dps_val: Comfort
+            value: comfort
+          - dps_val: ECO
+            value: eco
+          - dps_val: AntiFrost
+            value: away
+      - id: 101
+        type: integer
+        name: temperature
+        unit: C
+        range:
+          min: 10
+          max: 35
+  - entity: number
+    name: Eco Delta
+    icon: "mdi:leaf"
+    category: config
+    class: power
+    dps:
+      - id: 102
+        type: integer
+        name: value
+        unit: C
+        range:
+          min: 3
+          max: 7
+  - entity: number
+    name: AntiFrost
+    icon: "mdi:snowflake"
+    category: config
+    class: power
+    dps:
+      - id: 103
+        type: integer
+        name: value
+        unit: C
+        range:
+          min: 3
+          max: 7
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+  - entity: switch
+    name: Brightness
+    icon: "mdi:led-on"
+    translation_key: nightlight
+    category: config
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Auto off LCD
+    icon: "mdi:monitor"
+    category: config
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Sound
```

Comment:
What sound?

If it is beeps for user action, you can use `translation_key: keytone` instead of name and icon.